src/App.tsx

```diff
@@ -1,61 +1,110 @@
+import { useState } from 'react';
+
 import './App.scss';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+export const App: React.FC = () => {
+  const [curentTodos, setCurentTodos] = useState(todosFromServer);
+
+  const [newTitle, setNewTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [choosenUser, setChoosenUser] = useState(0);
+  const [hasUserError, setHasUserError] = useState(false);
+
+  const maxId = (arr: Todo[]): number => {
```

Comment:
please, give another name for this function. I think `generateNewId` would be great